infra/backup/README.md

```
@@ -0,0 +1,446 @@
+| 서버 로컬 | `~/gakkaweo/backups/` | 매일 백업 원본 | 7일 | `backup.sh` |
+| R2 | `daily/` | 매일 백업 | 60일 | R2 lifecycle 규칙 |
+| R2 | `weekly/` | 일요일 백업의 복사본 (주 1개) | 182일 | R2 lifecycle 규칙 |
+| R2 | `monthly/` | 매월 1일 백업의 복사본 (월 1개) | 730일 | R2 lifecycle 규칙 |
+
+암호화는 하지 않는다 (필요해지면 추후 추가).
+
+## 2. 구조
+
+### 실행 흐름
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)